### DISCUSSION TITLE
Resource quota and limits

### DISCUSSION THREAD

**Discussion by fisherking** [community/community#25104] [Actions]
<p>I see some resource limits for different plans on this page - <a href="https://github.com/features/actions" rel="nofollow noopener">https://github.com/features/actions</a></p>
<p>Is it possible to see resource quota usage for my organisation? I don’t see any page or specific API method with this information.</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>You could go to your organization settings page, click on Billing from the sidebar, Github actions minute quota is in Subscriptions tab.  </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/5143612035b4ca45c45a1efbe0f6f4493a7d4e76.png" data-download-href="/uploads/short-url/bAT29LRXZEVPtPnB048UjPjMVz8.png?dl=1" title="org settings.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181088782-5ea5d69a-dbbe-4767-a8d2-377cfe9e9e49.png" alt="org settings.png" data-base62-sha1="bAT29LRXZEVPtPnB048UjPjMVz8" width="537" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/5143612035b4ca45c45a1efbe0f6f4493a7d4e76_2_537x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/5143612035b4ca45c45a1efbe0f6f4493a7d4e76_2_805x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/5143612035b4ca45c45a1efbe0f6f4493a7d4e76.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/5143612035b4ca45c45a1efbe0f6f4493a7d4e76_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">org settings.png</span><span class="informations">986×918 66.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p><a href="https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhelp.github.com%2Fen%2Fgithub%2Fsetting-up-and-managing-billing-and-payments-on-github%2Fmanaging-your-spending-limit-for-github-actions%23managing-the-spending-limit-for-github-actions-for-your-organization&amp;data=02%7C01%7Cv-yjingz%40microsoft.com%7C477baadcf24c4306247e08d7883e03d9%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637127672245113735&amp;sdata=x%2Bns6%2BTBKGrMZ0o22Cnlgs8knsLqJNVxKMlCF1pceHQ%3D&amp;reserved=0" rel="nofollow noopener">https://help.github.com/en/github/setting-up-and-managing-billing-and-payments-on-github/managing-your-spending-limit-for-github-actions#managing-the-spending-limit-for-github-actions-for-your-organization</a></p>
